binary_tools.py

```diff
@@ -277,62 +346,34 @@ def binary_ephem(current_date, t0, T, smaj, ecc, inc, small_om, big_om, display=
     # # Period [years]
     # T          = 0.28479474332648874
 
-    # Beta Her
-    # Time of Periastron [years]
-    t0         = 1901.314794520548
-    # Semi-major axis [asec]
-    smaj       = 0.01137
-    # Eccentricity
-    ecc        = 0.55
-    # Inclination [degrees]
-    inc        = 53.8
-    # Argument of periastron
-    small_om   = 24.6
-    # Position angle of ascending node
-    big_om     = 341.9
-    # Period [years]
-    T          = 410.6/365.25
-
-    # Del Sge
-    # Time of Periastron [years]
-    t0         = 1979.93
-    # Semi-major axis [asec]
-    smaj       = 0.051
-    # Eccentricity
-    ecc        = 0.44
-    # Inclination [degrees]
-    inc        = 140
-    # Argument of periastron
-    small_om   = 257.7
-    # Position angle of ascending node
-    big_om     = 170.2
-    # Period [years]
-    T          = 10.11
-    
-    # epoch, x_pos, y_pos = binary_orbit(1000, smaj, ecc, inc, small_om, big_om, display = False)
+
+    ## Load Orbits params
+    t0, smaj, ecc, inc, small_om, big_om, T = get_Orbit_Param('Capella')
+
+    ## Orbit    
+    # epoch, x_pos, y_pos = binary_orbit(1000, t0, T, smaj, ecc, inc, small_om, big_om, display = False)
+
+
     date = Time('2021-03-19T00:00:00', format='isot')
     subaru = Observer.at_site("Subaru", timezone="US/Hawaii")
 
     ## Select date
     utc_time = subaru.datetime_to_astropy_time(date.datetime) # convert to UTC
     current_date = 2021.2123
+
+    ## Compute / display binary ephem
     sep, pa = binary_ephem(current_date, t0, T, smaj, ecc, inc, small_om, big_om, display=True)
-    ppp
+    
 
     ## Get the Parallactic Angle from astroplan library
     PA = get_Parallactic_Angle('Capella', 2020,9,16,4,48,32, display=True)
-    ## Get the Parallactic Angle from Telescope position
+    ## Get the Parallactic Angle from Telescope position # WARNING : IF Az comes from AO188 telemetry, need to subtract 180 degrees before using get_Parallactic_Angle_Subaru()
```

Comment:
In the class-like code, in the function get_Parallactic_Angle_Subaru, 180 degrees is substracted to the azimuth once again.
So it leads to a total of 360 degrees, so back to initial Az value. I removed both substraction: in the function and in the creation of the variable.
Consequently, such 180 degrees is no more needed. You could test with your class-like code (I did it hence I keep that simplification) to confirm that.